x/issuance/spec/05_params.md

```diff
@@ -0,0 +1,22 @@
+
+<!--
+order: 5
+-->
+
+# Parameters
+
+The issuance module has the following parameters:
+
+| Key        | Type           | Example       | Description                                 |
+|------------|----------------|---------------|---------------------------------------------|
+| Assets     | array (Asset)  | [{see below}] | array of assets created via issuance module |
```

Comment:
```suggestion
| Assets     | array (Asset)  | `[{see below}]` | array of assets created via issuance module |
```
This may need backticks for proper display in docs, might be safe since it's inside brackets